examples/tcp_connection.rs

```diff
@@ -48,15 +49,16 @@ fn main() {
         processor
     } = ClientProcessor::new();
 
+    let stats = Stats::new();
     // Initialize network communication
     let network = TcpStream::connect(&addr)
         .map_err(Error::from)
         .and_then(move |socket| {
             make_client_handshake(socket, &client_pk, &client_sk, &server_pk)
                 .map_err(Error::from)
         })
-        .and_then(|(socket, channel)| {
-            let secure_socket = Framed::new(socket, codec::Codec::new(channel));
+        .and_then(move |(socket, channel)| {
+            let secure_socket = Framed::new(socket, codec::Codec::new(channel, stats.clone()));
```

Comment:
No need to clone stats?

Reply:
Does not matter. You will remove this file in your PR